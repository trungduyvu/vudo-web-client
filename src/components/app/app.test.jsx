import renderer from 'react-test-renderer';

import React from 'react';
import App from './app';

describe('App component', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(<App />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
