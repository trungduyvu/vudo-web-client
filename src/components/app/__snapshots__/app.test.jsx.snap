// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly 1`] = `
<div>
  <header>
    <nav>
      <ul>
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/about"
            onClick={[Function]}
          >
            About
          </a>
        </li>
        <li>
          <a
            href="/register"
            onClick={[Function]}
          >
            Register
          </a>
        </li>
        <li>
          <a
            href="/sign-in"
            onClick={[Function]}
          >
            Sign In
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <main />
  <footer>
    Footer
  </footer>
</div>
`;
